src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -131,6 +166,54 @@ public void verifyTopLeaderboard() {
     verifyAdVisibleInSlot(presentLeaderboardSelector, presentLeaderboard);
   }
 
+  public void verifyFandomDesktopArticleTopLeaderboard() {
```

Comment:
What do you think about extracting all those changes to new class `AdsFandomBaseObject`? Then you don't need to type 'Fandom' in each method.